### ISSUE TITLE
Title Screen Glitch When Moving Sonic 

### ISSUE THREAD

**Issue by MilesTailsFan2005** [sonicretro/SonLVL#160]
I Tried Other Dissambelys I Mostly Use Hivebrain 68K And I Keep Having This Now On Any And Its Frustraiting
Only Happens On "Green Hill Act 1"

![Glitch](https://github.com/sonicretro/SonLVL/assets/52087744/7f12533d-423c-4673-b327-75d52e3f8d51)

**Comment by MilesTailsFan2005** [comment]
Yes I Deleted The Dissambily Folder And Redownloaded SonicLVL And Hivebrain 68K And Still Nothing

**Comment by Pr0jectFM** [comment]
This is a Sonic 1 issue, not a SonLVL issue
